src/shared/constants.ts

```diff
@@ -1,3 +1,5 @@
+import { COURSE_TITLES } from '../../dev-data/courseTitles.data';
```

Comment:
lets use alias import 
```suggestion
import { COURSE_TITLES } from 'data';
```